alibi/explainers/tests/test_simiarlity/test_grad_methods_integration.py

```diff
@@ -303,3 +334,147 @@ def test_multiple_test_instances_stored_grads_asym_dot(precompute_grads):
     scores = np.array([[50, 50, 2*50**2]], dtype=np.float32) / (denoms + 1e-7)
     scores.sort()
     np.testing.assert_allclose(explanation.scores, scores[:, ::-1], atol=1e-4)
+
+
+def test_non_trainable_layer_warnings_tf():
+    """Test Non trainable layer warnings tensorflow.
```

Comment:
Non trainable -> non-trainable?
I would suggest to follow the docstring conventions for the tests too: full name of the explainer, proper use of single/double ticks etc. I know this won't show up anywhere, but ...